ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -802,6 +802,68 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure Siri is enabled (Based on organization's policy) (MDM Required)
```

Comment:
When enabled, there are other settings they recommend the admin sets:
```
{
    LockscreenEnabled = 0/1;
    StatusMenuVisible = 0/1;
    TypeToSiriEnabled = 0/1;
    VoiceTriggerUserEnabled = 0/1;
}
```

Reply:
See additional queries below.